faktory_exporter.go

```diff
@@ -134,12 +135,13 @@ func New(faktoryURL string) (*Exporter, error) {
 			Name:      "total",
 			Help:      "Total amount of jobs.",
 		}, []string{"status"}),
-		queues: prometheus.NewGauge(prometheus.GaugeOpts{
+		totalQueues: prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
I'm wondering if for better backward compatibility (if we're planning to make an upstream PR), we rather keep `queues`, and add something like `queue_sizes`

Reply:
hmm, no output changes, just the code is more factual.

Reply:
Yeah, best not to break existing things if we can avoid it.

Reply:
Ah, I see, I misunderstood how the metric names are generated. 👍 from me

Reply:
Ah, I see, I misunderstood how the metric names are generated. 👍 from me